### PR TITLE
Finds the Magnitude of a Vector

### PR DESCRIPTION
**Pull request by Shreyas-OwO**
Hello @andrefpoliveira!

This program is a contribution to the collection of the example programs is the  ``examples`` directory.

This program defines a function which calculates and returns the magnitude of a vector whose direction ratios are put in a list and passed to it (the defined function).

Also, it would be helpful to me if you could add ``hacktoberfest-accepted`` label to this PR.

Thanks! :smile:

### CODE REVIEW

File: examples/VectorMagnitude/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Vector Magnitude
+
+Arguments:
+```
+Argument #1 - Type: Integer/Float
```
Comment (andrefpoliveira) [inline examples/VectorMagnitude/README.md:5]: @Shreyas-OwO first of all, thank you for your contribution!
This first argument is actually a list of integer/floats 😄 Could you please fix it please?

**Review by andrefpoliveira — CHANGES_REQUESTED** [review]

File: examples/VectorMagnitude/README.md (hunk 1 above)
Comment (Shreyas-OwO) [reply examples/VectorMagnitude/README.md]: Done!

**Comment by Shreyas-OwO** [comment]
Hello @andrefpoliveira! The suggested changes have been made.

**Review by andrefpoliveira — APPROVED** [review]
Thanks!!

**Pull request merged**